### PR TITLE
Adds descriptions to library manuals

### LINKED ISSUE #18378: Book descriptions are messed up
**Issue Description**:

Some books (at the very least the corporate manuals) have error messages in their descriptions.

**What did you expect to happen**: 

Proper descriptions.

**What happened instead**:

dreamseeker_AW44MYpHQg

**Why is this bad/What are the consequences**:

This is not an IC message where it should be.

**Steps to reproduce the problem**:

1. Click library computer
2. Go to the corporate section where you print pre-made manuals such as the SOP books
3. Print some
4. Examine them

**When did the problem start happening**:

Probably library rework?

**Extra information**:

the description told me to make this report and I am powerless to resist it

### PR DESCRIPTION
**Pull request by Shadow-Quill**
## What Does This PR Do
Adds descriptions to all the pre-made manuals, so that the "You shouldn't see this!!!" message is vaporized.
Fixes #18378.

## Why It's Good For The Game
People aren't told to tell coders about manuals not having a description.

## Changelog
:cl:
fix: Added descriptions to pre-made manuals, which prevents the "Please make a report on the github" message from showing.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/manuals.dm [hunk 1]
```diff
@@ -662,6 +673,7 @@
 
 /obj/item/book/manual/hydroponics_pod_people
 	name = "The Human Harvest - From seed to market"
+	desc = "Burry pictures of people coming out of pods are taped to the cover."
```
Comment (Contrabang) [inline code/game/objects/items/weapons/manuals.dm:676]: Blurry?

File: code/game/objects/items/weapons/manuals.dm [hunk 2]
```diff
@@ -743,34 +757,39 @@
...
 	name = "Station Repairs and Construction"
+	desc = "A guide on ow to fix things without duct tape."
 	icon_state ="bookEngineering"
 	author = "Engineering Encyclopedia"
 	title = "Station Repairs and Construction"
 	wiki_article_title = "Guide_to_Construction"
 
 /obj/item/book/manual/wiki/faxes
 	name = "Guide to Faxes"
+	desc = "NanoTrasen's own manual on how to write faxefs."
```
Comment (Contrabang) [inline code/game/objects/items/weapons/manuals.dm:792]: Faxes?

**Comment by Contrabang** [comment]
Looks pretty good to me otherwise, nice and clever descriptions.

File: code/game/objects/items/weapons/manuals.dm (hunk 1 above)
Comment (Shadow-Quill) [reply code/game/objects/items/weapons/manuals.dm:676]: I can type, I swear.

**Comment by Sirryan2002** [comment]
Thank you, I totally forgot these, you're also welcome to update whatever you want in the manuals as some aren't 100% up to date or could be better.

File: code/game/objects/items/weapons/manuals.dm [hunk 3]
```diff
@@ -743,34 +757,39 @@
...
 	name = "Mentoring your Experiments"
+	desc = "A madman's ramblings on how to experiment with the E.X.P.E.R.I-MENTOR."
 	icon_state = "rdbook"
 	author = "Dr. H.P. Kritz"
 	title = "Mentoring your Experiments"
 	wiki_article_title = "Guide_to_the_E.X.P.E.R.I-MENTOR"
 
 /obj/item/book/manual/wiki/chef_recipes
 	name = "Chef Recipes"
+	desc = "Knives, Ovens, and You:  A guide to cooking."
```
Comment (S34NW) [inline code/game/objects/items/weapons/manuals.dm:776]:
```suggestion
	desc = "Knives, Ovens, and You: A guide to cooking."
```

File: code/game/objects/items/weapons/manuals.dm [hunk 4]
```diff
@@ -195,6 +198,7 @@
 
 /obj/item/book/manual/atmospipes
 	name = "Pipes and You: Getting To Know Your Scary Tools"
+	desc = "A plumbers guide on how to efficiently plumb and clean out old drone shells."
```
Comment (S34NW) [inline code/game/objects/items/weapons/manuals.dm:201]:
```suggestion
	desc = "A plumber's guide on how to efficiently plumb and clean out old drone shells."
```

**Review by S34NW — COMMENTED** [review]
Good overall, minor grammar stuff

**Review by S34NW — APPROVED** [review]

File: code/game/objects/items/weapons/manuals.dm [hunk 5]
```diff
@@ -743,34 +757,39 @@
...
 	name = "Chef Recipes"
+	desc = "Knives, Ovens, and You: A guide to cooking."
 	icon_state = "cook_book"
 	author = "NanoTrasen"
 	title = "Chef Recipes"
 	wiki_article_title = "Guide_to_Food_and_Drinks#Food"
 
 /obj/item/book/manual/wiki/engineering_construction
 	name = "Station Repairs and Construction"
+	desc = "A guide on ow to fix things without duct tape."
```
Comment (Sirryan2002) [inline code/game/objects/items/weapons/manuals.dm:784]:
```suggestion
	desc = "A guide on how to fix things without duct tape."
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**